frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/common/FetchUpToDateTimeoutStatus.java

```diff
@@ -6,6 +6,7 @@ public final class FetchUpToDateTimeoutStatus {
   private FetchUpToDateTimeoutStatus() {}
 
   public static final long FETCH_ALL_UP_TO_DATE_TIMEOUT_MILLIS = 60 * 1000;
+  public static final String FETCH_ALL_UP_TO_DATE_TIME_UNIT = "a minute";
```

Comment:
Nit: `TIME_UNIT` isn't the most fortunate name... imagine the timeout is 180 seconds... then the "time unit" would be `three minutes`... which isn't a time _unit_ as such (a `minute` or a `week` is, but not `three minutes`) 🤔 

I'd suggest just
```suggestion
  public static final String FETCH_ALL_UP_TO_DATE_TIMEOUT_AS_STRING = "a minute";
```